docs/cuopt/source/conf.py

```
@@ -279,51 +276,60 @@
+
 def skip_unwanted_inherited_members(app, what, name, obj, skip, options):
-    inherited_to_skip = {"as_integer_ratio", "conjugate", "from_bytes", "to_bytes", "is_integer", "bit_count", "bit_length", "is_integer"}  # add more as needed
+    inherited_to_skip = {
+        "as_integer_ratio",
+        "conjugate",
+        "from_bytes",
+        "to_bytes",
+        "is_integer",
+        "bit_count",
+        "bit_length",
+        "is_integer",
```

Ruff (0.14.3)

Sets should not contain duplicate item `"is_integer"`

Remove duplicate item

(B033)